static/javascripts/components/survey_card.jsx

```diff
@@ -73,9 +101,8 @@ var SurveysPage = React.createClass({
             item_title: this.state.item_title,
             questions : questions
         };
-        console.log(surveyObj);
-
 
+        var test = this.checkSurvey(surveyObj);
```

Comment:
Does this even stop the ajax from posting? I might not know how alerts work but wouldn't we want to say if(test==true){ajax post}?